### ISSUE TITLE
BSLegacyUtil cannot find directory path when installing legacy version

### ISSUE THREAD

**Issue by mazina** [MintLily/BSLegacyUtil#4]
I previously used your utility to successfully install a legacy version instead (I think 1.16.4 instead of 1.17).  When Beatsaber auto-updated to 1.18, my install would no longer launch saying some files were mismatched.  I verified the files, which led to 75 files being replaced and 1.18 launching properly again.  I tried to revert bacj to 1.17 or 1.16 (using BSLegacyUtil on the command line) but cannot.  I am able to download previous versions, but the utility crashes when I try to install them.  I get the following error:
```
[BSLegacyUtil] [Error] ====== STACK =====
   at Microsoft.VisualBasic.FileIO.FileSystem.CopyOrMoveDirectory(CopyOrMove operation, String sourceDirectoryName, String destinationDirectoryName, Boolean overwrite, UIOptionInternal showUI, UICancelOption onUserCancel)
   at Microsoft.VisualBasic.FileIO.FileSystem.MoveDirectory(String sourceDirectoryName, String destinationDirectoryName, Boolean overwrite)
   at BSLegacyUtil.Functions.Mod.modGame(String gameVersion)
   at BSLegacyUtil.Program.SelectGameVersion(Boolean dlGame)
   at BSLegacyUtil.Program.BeginInputOption()
   at BSLegacyUtil.Functions.Download.DLGame(String gameVersionInput)
 ===== ERROR =====
System.IO.DirectoryNotFoundException: Could not find directory 'Resources\Beat Saber_Data'.
   at Microsoft.VisualBasic.FileIO.FileSystem.CopyOrMoveDirectory(CopyOrMove operation, String sourceDirectoryName, String destinationDirectoryName, Boolean overwrite, UIOptionInternal showUI, UICancelOption onUserCancel)
   at Microsoft.VisualBasic.FileIO.FileSystem.MoveDirectory(String sourceDirectoryName, String destinationDirectoryName, Boolean overwrite)
   at BSLegacyUtil.Functions.Mod.modGame(String gameVersion)
   at BSLegacyUtil.Program.SelectGameVersion(Boolean dlGame)
   at BSLegacyUtil.Program.BeginInputOption()
   at BSLegacyUtil.Functions.Download.DLGame(String gameVersionInput)
```

Any idea what could be going wrong?

**Comment by MintLily** [comment]
It seems that a function is broken, and I forgot to remove it; so that's my bad.
I have fixed this in version [v2.2.1](https://github.com/MintLily/BSLegacyUtil/releases/tag/2.2.1). And since you're modding a modern version, you can simply run the exe of Mod Assistant, if the Resources folder or from the [Mod Assistant github](https://github.com/Assistant/ModAssistant)

**Comment by mazina** [comment]
Thanks for checking this out so quickly!
